index.html

```diff
@@ -0,0 +1,287 @@
+                        </div>
+                        <div class="people-count">
+                            <img src="./Images/building.png" alt="building Image" style="margin-top: 7px; width: 20px; height: 20px;">
+                            <p class="people">24 people going</p>
+                        </div>
+                    </div>
+                    <div class="floating-card">
+                        <div class="img-wrapper">
+                            <img src="./Images/mahal.png" alt="Europe Image">
+                        </div>
+                        <div class="ongoing">
+                            <p>Ongoing</p>
+                            <p class="trip">Trip to rome</p>
+                        </div>
+
+                        <div class="percentage">
+                            <div class="first-line">
+                                <p>40%</p>
+                                <p>completed</p>
+                            </div>
+                            <div class="progress-bar">
+                                <div class="filled"></div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="heart">
+                        <img src="./Images/heart.png" alt="Heart Image">
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!-- Booking Section ends-->
+        
+
+        <!-- Subscribe section starts -->
+        <div class="subscribe-section">
+            <h2>Subscribe to get information, latest news and other interesting offers about cobham</h2>
+            <div class="input">
+                <input type="mail" placeholder="Your email">
+                <button>Subscribe</button>
+            </div>
+        </div>
+        <!-- subscribe section ends -->
+
+
+        <div class="send-newsletter">
+            <img src="./Images/newsletter.png" alt="send image">
+        </div>
+        <div class="spiral-left">
+            <img src="./Images/spiral.png" alt="spiral image">
+        </div>
+        <div class="spiral-right">
+            <img src="./Images/spiral.png" alt="spiral image">
+        </div>
+
+        <div class="plus-pattern-down">
+            <img src="./Images/plus.png" alt="design image">
+        </div>
+
+
+        <footer>
+            <div class="footer">
+                <div class="company-name">
+                    <div class="title">TravelPro</div>
+                    <div class="description">Book your trip in minute, get full Control for much loner.</div>
+                </div>
+                <div class="company">
+                    <ul>
+                        <li>Company</li>
+                        <li>About</li>
+                        <li>Careers</li>
+                        <li>Mobile</li>
+                    </ul>
+                </div>
+                <div class="contact">

```

Comment:
```suggestion
                <div class="contact">

```

if you didn't declare the class name why u declare in html is there any reason ?